pontoon/base/urls.py

```diff
@@ -125,6 +125,8 @@
         name='pontoon.transifex'),
     url(r'^csrf/$', views.get_csrf,
         name='pontoon.csrf'),
+    url(r'^project-details/$', views.get_project_details,
```

Comment:
I think `projects/slug/details/` is a more sensible URL for this, and lets you avoid having a try block in the view just to catch the missing parameter.